build_debian.sh

```diff
@@ -300,6 +300,10 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT easy_install pip
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install 'docker-py==1.6.0'
 ## Note: keep pip installed for maintainance purpose
 
+## Get gcc and python dev pkgs
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install gcc libpython2.7-dev
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install 'netifaces==0.10.6'
```

Comment:
sonic-utilities package is also installed in docker-sonic-vs, so can you also install the netifaces in the docker vs. You need to add this in platform/vs/docker-sonic-vs/Dockerfile.j2.

Reply:
Not needed. netifaces is in docker-sonic-vs